### PR TITLE
Fix for previous output not being quantized

### PR DESCRIPTION
**Pull request by jecorona97**

### CODE REVIEW

**Comment by zhuangh** [comment]
@jecorona97 test failed.

cc @lishanok , I cannot assign you to the reviewers, could you do that by yourself :)

File: qkeras/qrecurrent.py [hunk 1]
```diff
@@ -75,22 +75,26 @@ def __init__(self,
                kernel_quantizer=None,
                recurrent_quantizer=None,
                bias_quantizer=None,
+               state_quantizer=None,
```
Comment (lishanok) [inline qkeras/qrecurrent.py:78]: Please apply these changes to other types of RNNs as well

File: qkeras/qrecurrent.py [hunk 2]
```diff
@@ -161,14 +170,14 @@ def call(self, inputs, states, training=None):
...
     if rec_dp_mask is not None:
-      prev_output = prev_output * rec_dp_mask
+      quantized_prev_output = quantized_prev_output * rec_dp_mask
 
     if self.recurrent_quantizer:
       quantized_recurrent = self.recurrent_quantizer_internal(self.recurrent_kernel)
     else:
       quantized_recurrent = self.recurrent_kernel
 
-    output = h + K.dot(prev_output, quantized_recurrent)
```
Comment (lishanok) [inline qkeras/qrecurrent.py:171]: does the output include both h and c states?
Does it make sense to set the same quantizer to both h and c state?

Comment (jecorona97) [reply qkeras/qrecurrent.py:171]: Just separated them in the last set of commits :)

File: qkeras/qrecurrent.py (hunk 1 above)
Comment (jecorona97) [reply qkeras/qrecurrent.py:78]: Just pushed the QSimpleRNN and QLSTM fix. Please let me know what you think. Both networks seem to learn with some performance degradation and higher sensitivity to initialization. If you think it looks good, I'll go ahead and work on the necessary AutoQKeras changes and update the notebooks.

Comment (jecorona97) [reply qkeras/qrecurrent.py:78]: As well as the GRU fix :)

**Comment by zhuangh** [comment]
Hi, any news on this @jecorona97 thanks

**Comment by zhuangh** [comment]
courtesy ping @jecorona97

**Comment by zhuangh** [comment]
https://github.com/google/qkeras/commit/67e7c6b8cbd6befd594f142187ac4b73b35512ac is merged by @lishanok Thank you! close this PR.

**Pull request closed without merging**